templates/includes/records/record-details-community.html

```diff
@@ -78,4 +78,25 @@
         <th scope="row">{{ 'type'|as_label }}</th>
         <td>{{ record.type }}</td>
     </tr>
-{% endif %}
\ No newline at end of file
+{% endif %}
+{% if record.provenance %}
+    <tr>
+        <th scope="row">{{ 'provenance'|as_label }}</th>
+        <td>{{ record.provenance }}</td>
+    </tr>
+{% endif %}
+{% if record.place_of_deposit %}
+    <tr>
+        <th scope="row">{{ 'place_of_deposit'|as_label }}</th>
```

Comment:
I've raised a query in the ticket - as per the scope of BE attribute for this one - having Label(Key) + Value dynamic. (updated)

Reply:
`place of deposit` is now `repository` as discussed in ticket.